src/report/business-reports/aging.scm

```diff
@@ -319,7 +331,7 @@ more than one currency. This report is not designed to cope with this possibilit
     (qof-query-set-sort-increasing query #t #t #t)))
      
 
-(define (aging-options-generator options)
+(define (aging-options-generator options receivable)
```

Comment:
I can see you need a way to differentiate between receivable and payable reports. However I'm a bit reluctant to accept additional function parameters if there's another way to pass the information.
I believe in this case it would be cleaner to define an internal option (which means invisible to the user) instead of an extra parameter. This internal option would become part of the options parameter and can be queried like any other option in the remainder of the code.